scripts/parachain/local.sh

```diff
@@ -6,9 +6,9 @@
 
 set -euxo pipefail
 
-PARACHAIN_CHAIN=battery_park_staging
+PARACHAIN_CHAIN=battery_station_staging
 PARACHAIN_ID=9123
-POLKADOT_BRANCH=release-v0.9.4
+POLKADOT_BRANCH=release-v0.9.7
```

Comment:
Should this be v0.9.7?

Reply:
`v0.9.7` because Rococo is probably using this version

Reply:
But this is the local environment, isn't it? If I understand correctly, we launch the Rococo chain locally "rococo-local". This way or another, I don't mind if it works properly.

Reply:
The branch version is mirroring a production environment the same way `PARACHAIN_CHAIN` is `battery_station_staging` instead of `dev`.

Changed back to 0.9.6

Reply:
Thanks for the clarification.